.travis.yml

```diff
@@ -23,6 +25,8 @@ matrix:
       env: RSPEC_VERSION=2.14
     - rvm: 1.8.7
       env: PUPPET_VERSION=4.0
+    - rvm: 1.8.7
+      env: PUPPET_VERSION=4.1
```

Comment:
> oooh! i didn't know that puppet worked with ruby 2.2 already!

Just to be pedantic: I'd apply `s/worked with/is spec tested with/`  :)

Less pedantically, it *is* more lightly tested against 2.2, so there may be issues we discover as it gets used more widely in ruby 2.2 environments.